Suggested routes inconsistencies

NayanaKamtekar opened the pull request:
I have changed suggested_routes table foreign key.  As mentioned in the task routers, controllers and others should be also called suggested_routes.  But you already suggested changes to ABOD in his PR. I hope he will do that.

NayanaKamtekar committed:
Added port.router and port controller for test

NayanaKamtekar committed:
Missing files

zkwsk committed:
Fix authorizeUser.

zkwsk committed:
Add missing export.

NayanaKamtekar committed:
Merge branch 'develop' of https://github.com/HackYourFuture-CPH/ov-class09-fp into develop

NayanaKamtekar committed:
Merge branch 'develop' of https://github.com/HackYourFuture-CPH/ov-class09-fp into develop

NayanaKamtekar committed:
Merge branch 'develop' of https://github.com/HackYourFuture-CPH/ov-class09-fp into develop

NayanaKamtekar committed:
Merge branch 'develop' of https://github.com/HackYourFuture-CPH/ov-class09-fp into develop

NayanaKamtekar committed:
Merge branch 'develop' of https://github.com/HackYourFuture-CPH/ov-class09-fp into develop

NayanaKamtekar committed:
changed foreign key

NayanaKamtekar committed:
suggested_routes inconsistencies: changed foreign key

zkwsk commented on db/Schema.dbml:
```diff
@@ -135,8 +136,8 @@ table waypoints {
-  created_on timestamp [not null, default: `now()`]
-  updated_on timestamp [not null, default: `now()`, note: 'TODO: Add ON UPDATE ON UPDATE CURRENT_TIMESTAMP']
+  created_at timestamp [not null, default: `now()`]
+  updated_at timestamp [not null, default: `now()`, note: 'TODO: Add ON UPDATE ON UPDATE CURRENT_TIMESTAMP']
```
Really nice that you are finally changing the `created_on` to `created_at`. Remember you need to update the db diagram as well. Refer to the instructions in the readme on how to.

NayanaKamtekar replied:
I have modified db diagram as well.

zkwsk commented on src/server/migrations/20190918222221_suggested_routes_foreign_key.js:
```diff
@@ -0,0 +1,23 @@
+      .onDelete("CASCADE")
+      .onUpdate("RESTRICT");
+  });
+};
```
Can you please explain this migration? I don't see it perform any changes to my schema for `suggested_routes`?

NayanaKamtekar replied:
It just changes foreign key name from routes_voyage_id_foreign to suggested_routes_voyage_id_foreign

zkwsk commented:
Also. Could you please clone the repo directly from the main fork and commit from there from now on? :)

NayanaKamtekar committed:
Modified Schema.dbml and Schema.png

NayanaKamtekar committed:
Modified Schema.dbml and Schema.png

bytewiz reviewed (dismissed):
Looks good to me 👍

NayanaKamtekar committed:
Delete 20190919224640_voyages_foreign_keys.js

This file is not a part of suggested-routes-inconsistencies branch . This file should be in voyages-foreign-keys branch.

zkwsk commented:
Merged this based on the fact that @bytewiz had approved it previously :)
